From 0515336a0095fb9a1046eeffd94d0220669d779e Mon Sep 17 00:00:00 2001
From: "kalyan.kondapally" <kalyan.kondapally@intel.com>
Date: Mon, 19 Jan 2015 07:50:47 -0800
Subject: [PATCH] Ozone: Build fix on Linux.

Clang complains about un-used funtions on Linux builds. This CL fixes
it.

Review URL: https://codereview.chromium.org/841113004

Cr-Commit-Position: refs/heads/master@{#312121}
---
 ui/events/ozone/layout/xkb/xkb_keyboard_layout_engine.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ui/events/ozone/layout/xkb/xkb_keyboard_layout_engine.cc b/ui/events/ozone/layout/xkb/xkb_keyboard_layout_engine.cc
index ba4acf0..bf7d158 100644
--- a/ui/events/ozone/layout/xkb/xkb_keyboard_layout_engine.cc
+++ b/ui/events/ozone/layout/xkb/xkb_keyboard_layout_engine.cc
@@ -626,6 +626,7 @@ const PrintableSimpleEntry kSimpleMap[] = {
     {0x0259, VKEY_OEM_3},      // schwa
 };
 
+#if defined(OS_CHROMEOS)
 void LoadKeymap(const std::string& layout_name,
                 scoped_refptr<base::SingleThreadTaskRunner> reply_runner,
                 const LoadKeymapCallback& reply_callback) {
@@ -653,6 +654,7 @@ void LoadKeymap(const std::string& layout_name,
     LOG(ERROR) << "Keymap file failed to load: " << layout_name;
   }
 }
+#endif
 
 }  // anonymous namespace
 
-- 
2.1.4

